pubspec.lock

```diff
@@ -35,7 +42,7 @@ packages:
       name: collection
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.14.12"
+    version: "1.15.0-nullsafety"
```

Comment:
Either you or I have the wrong setup because I had to run `flutter pub get` again after your last PR and it didn't produce the same pubspec.lock. But it's okay for now